docs/source/deployment/prefect.md

```diff
@@ -12,9 +12,9 @@ Prefect Server ships out-of-the-box with a fully featured user interface.
 
 To use Prefect Core and Prefect Server, make sure you have the following prerequisites in place:
```

Comment:
To use Prefect Core and Prefect Server, ensure you have the following prerequisites: